README.md

```diff
@@ -195,14 +195,26 @@ altTextAndImages :: Scraper String [(String, URL)]
 altTextAndImages =
     -- 1. First narrow the current context to each img tag.
     chroots "img" $ do
-        -- 2. Use Any to access all the relevant content from the the currently
+        -- 2. Use anySelector to access all the relevant content from the the currently
         -- selected img tag.
-        altText <- attr "alt" Any
-        srcUrl  <- attr "src" Any
+        altText <- attr "alt" anySelector
+        srcUrl  <- attr "src" anySelector
         -- 3. Combine the retrieved content into the desired final result.
         return (altText, srcUrl)
 ```
 
 For the full source of this example, see
 [generalized-repetition](https://github.com/fimad/scalpel/tree/master/examples/generalized-repetition/)
 in the examples directory.
+
+### Matching wildcards
+
+ - `anySelector` will match all tags
```

Comment:
I think this section should go under the OverloadedStrings section above since it's a bit more niche than Overloaded strings, but more basic than the chroots stuff.

Also, I think we could expand this a bit to give readers a bit more context and example snippets of how each are used. For example, something like:

### Matching Wildcards

Scalpel has 3 different wildcard values each corresponding to a distinct use case.

* If you want to match all tags, then use `anySelector`.

```haskell
idsOfAllTags = attrs "id" anySelector
```

* If you don't care about the name of the tag but want to include attribute constraints, then use `AnyTag`.

```haskell
idsOfTagsWithClassButton = attrs "id" $ AnyTag @: [hasClass "button"]
```

* If you want to constrain based on the value of an attribute, but you don't care what the name of the attribute is, use `AnyAttribute`.

```haskell
idsOfTagsWithAnAttributeWhoseValueIsButton = attrs "id" $ AnyTag @: [AnyAttribute @= "button"]
```